test/general/TestItems.py

```diff
@@ -44,8 +44,6 @@ def testItemNameGroupConflict(self):
     def testItemCountGreaterEqualLocations(self):
         for game_name, world_type in AutoWorldRegister.world_types.items():
 
-            if game_name in {"Final Fantasy"}:
-                continue
```

Comment:
always leaking stuff :D

Reply:
odd i dropped that locally. might need to force push.